utility/importer/chrome_importer.cc

```diff
@@ -86,11 +86,7 @@ void ChromeImporter::StartImport(const importer::SourceProfile& source_profile,
     bridge_->NotifyItemEnded(importer::PASSWORDS);
   }
 
-  if ((items & importer::COOKIES) && !cancelled()) {
-    bridge_->NotifyItemStarted(importer::COOKIES);
-    ImportCookies();
```

Comment:
seems like you forgot to remove https://github.com/brave/brave-core/blob/b0bd76cc949b003697f5ab1b98307cf93ca349bb/utility/importer/chrome_importer.cc#L347

Reply:
It is still in use by `BraveImporter` - asked about it in the ticket.

Reply:
ah, right, `BraveImporter`